.pipelines/PowerShell-Coordinated_Packages-Official.yml

```
@@ -0,0 +1,270 @@
+        binskim:
+          enabled: false
+        apiscan:
+          enabled: false
+        tsaOptionsFile: .config\tsaoptions.json
+
+    stages:
+    - stage: prep
+      jobs:
+      - template: /.pipelines/templates/checkAzureContainer.yml@self
+    - stage: macos
+      dependsOn: ['prep']
+      jobs:
+      - template: /.pipelines/templates/mac.yml@self
+        parameters:
+          buildArchitecture: x64
+      - template: /.pipelines/templates/mac.yml@self
+        parameters:
+          buildArchitecture: arm64
+    - stage: linux
+      dependsOn: ['prep']
+      jobs:
+      - template: /.pipelines/templates/build-linux.yml@self
+        parameters:
+          Runtime: 'linux-x64'
+          JobName: 'linux_x64'
+
+      - template: /.pipelines/templates/build-linux.yml@self
+        parameters:
+          Runtime: 'linux-x64'
+          JobName: 'linux_x64_minSize'
+          BuildConfiguration: 'minSize'
+
+      - template: /.pipelines/templates/build-linux.yml@self
+        parameters:
+          Runtime: 'linux-arm'
+          JobName: 'linux_arm'
+
+      - template: /.pipelines/templates/build-linux.yml@self
+        parameters:
+          Runtime: 'linux-arm64'
+          JobName: 'linux_arm64'
+
+      - template: /.pipelines/templates/build-linux.yml@self
+        parameters:
+          Runtime: 'fxdependent-linux-x64'
+          JobName: 'linux_fxd_x64_mariner'
+
+      - template: /.pipelines/templates/build-linux.yml@self
+        parameters:
+          Runtime: 'fxdependent-linux-arm64'
+          JobName: 'linux_fxd_arm64_mariner'
+
+      - template: /.pipelines/templates/build-linux.yml@self
+        parameters:
+          Runtime: 'fxdependent-noopt-linux-musl-x64'
+          JobName: 'linux_fxd_x64_alpine'
+
+      - template: /.pipelines/templates/build-linux.yml@self
+        parameters:
+          Runtime: 'fxdependent'
+          JobName: 'linux_fxd'
+
+      - template: /.pipelines/templates/build-linux.yml@self
+        parameters:
+          Runtime: 'linux-musl-x64'
+          JobName: 'linux_x64_alpine'
+
+      # - template: /.pipelines/templates/linux.yml@self
+      #   parameters:
+      #     buildName: deb
+      # - template: /.pipelines/templates/linux.yml@self
+      #   parameters:
+      #     buildName: rpm
+      #     parentJob: build_deb
+      # - template: /.pipelines/templates/linux.yml@self
+      #   parameters:
+      #     buildName: fxdependent
+      #     parentJob: build_deb
+      # - template: /.pipelines/templates/linux.yml@self
+      #   parameters:
+      #     buildName: alpine
+    - stage: windows
+      dependsOn: ['prep']
+      jobs:
+      - template: /.pipelines/templates/windows-hosted-build.yml@self
+        parameters:
+          Architecture: x64
+          BuildConfiguration: release
+          JobName: build_windows_x64_release
+      - template: /.pipelines/templates/windows-hosted-build.yml@self
```

```suggestion

      - template: /.pipelines/templates/windows-hosted-build.yml@self
```
Can we add a new line between jobs?  I find it a lot easier to read.

---

done